sailor/utils/oauth_wrapper/OAuthServiceImpl.py

```diff
@@ -88,7 +92,8 @@ def request(self, method, url, **req_kwargs):
         LOG.debug('Calling %s', url)
         response = session.request(method, url, **req_kwargs)
         if response.ok:
-            if response.headers['Content-Type'] == 'application/json':
+            r_headers = {k.lower(): v for (k, v) in response.headers.items()}
```

Comment:
you don't need to manually make the keys lower for the `r_headers`, the `get` is overwritten and non-sensitive automatically

Reply:
Done, as discussed. Btw. I've double-checked what requests lib is doing.

They are using a special CaseInsensitiveDict which stores the header keys in the dict lowercase and also makes the lookup lower case automatically.